pkg/logs/tailers/socket/tailer.go

```diff
@@ -71,6 +71,7 @@ func (t *Tailer) forwardMessages() {
 		if len(output.GetContent()) > 0 {
 			origin := message.NewOrigin(t.source)
 			origin.SetTags(output.ParsingExtra.Tags)
+			origin.LogSource.RecordBytes(int64(len(output.GetContent())))
```

Comment:
Thanks for this change!

One small improvement - Can we move this down to line `101` as:
`t.source.RecordBytes(int64(len(data)))`

The problem with doing this here, is it only counts bytes after the decoder processed the log. So if a regex/exclusion rule was applied the bytes would not be counted.